zio-http/src/main/scala/zhttp/http/Headers.scala

```diff
@@ -26,6 +26,8 @@ final case class Headers(toChunk: Chunk[Header]) extends HeaderExtension[Headers
 
   def toList: List[(String, String)] = toChunk.map { case (name, value) => (name.toString, value.toString) }.toList
 
+  def modifyHeaders(f: Header => Header): Headers = Headers(toChunk.map(f(_)))
```

Comment:
```suggestion
  def modify(f: Header => Header): Headers = Headers(toChunk.map(f(_)))
```

Let's keep it as `modify` because we know it's applicable only on headers.